nemo_skills/inference/server/model.py

```diff
@@ -78,6 +79,7 @@ def __init__(
         max_code_output_characters=1000,
         code_execution_timeout=10.0,
         max_code_executions=3,
+        error_recovery_attempts=0,
```

Comment:
Can you please add the documentation?

Reply:
Also can you please add some logic to NeMo model that will error out when this new parameter is used (as well as stop_on execution error and other things that are only supported in external loop mode)? So that people don't try to use them and be surprised that they don't work, until we add the logic to handle it.